framework/moose.mk

```diff
@@ -28,6 +28,19 @@ hit_LIB       := $(hit_DIR)/libhit-$(METHOD).la
 hit_deps      := $(patsubst %.cc, %.$(obj-suffix).d, $(hit_srcfiles))
 
 #
+# hit python bindings
+#
+pyhit_srcfiles  := $(hit_DIR)/hit.cpp $(hit_DIR)/lex.cc $(hit_DIR)/parse.cc
+pyhit_LIB       := $(hit_DIR)/hit.so
+
+# the sed hack here is to get around a bug in python's distutils that only uses the first word of
```

Comment:
What sed hack?

Reply:
I removed the hack - I'll remove the comment.